docker/Dockerfile

```diff
@@ -1,6 +1,7 @@
 ARG PYTORCH="1.6.0"
 ARG CUDA="10.1"
 ARG CUDNN="7"
+ARG MMCV="1.3.12"
 
 FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-devel
 
```

Comment:
Hello.When I try to build docker image
/docker$ docker build -t mmseg .
I get a error
Error log:
Step 10/16 : RUN conda clean --all
 ---> Using cache
 ---> 025c6545676b
Step 11/16 : RUN ["/bin/bash", "-c", "pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html"]
 ---> Running in 24b4e9123299
Looking in links: https://download.openmmlab.com/mmcv/dist/cu/torch/index.html
ERROR: Could not find a version that satisfies the requirement mmcv-full== (from versions: 1.0rc0, 1.0rc2, 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.1.6, 1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.2.6, 1.2.7, 1.3.0, 1.3.1, 1.3.3, 1.3.4, 1.3.5, 1.3.6, 1.3.7, 1.3.8, 1.3.9, 1.3.10, 1.3.11, 1.3.12)
ERROR: No matching distribution found for mmcv-full==
The command '/bin/bash -c pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html' returned a non-zero code: 1

And we can see "Looking in links: https://download.openmmlab.com/mmcv/dist/cu/torch/index.html",script cannt get args ${MMCV},${CUDA//./},${PYTORCH},beacause args workspace.You can see https://yeasy.gitbook.io/docker_practice/image/dockerfile/arg

We can fix dockerfile to resolve problem:

ARG PYTORCH="1.6.0"
ARG CUDA="10.1"
ARG CUDNN="7"

FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-devel

ARG PYTORCH="1.6.0"
ARG CUDA="10.1"
ARG CUDNN="7"
ARG MMCV="1.3.12"